src/LocaleString.res

```diff
@@ -111,9 +111,9 @@ let defaultLocale = {
   useExisitingSavedCards: "Use saved debit/credit cards",
   saveCardDetails: "Save card details",
   addBankAccount: "Add bank account",
-  achBankDebitTerms: _str =>
+  achBankDebitTerms: _ =>
     `Your ACH Debit Authorization will be set up now, but we'll confirm the amount and let you know before future payments are taken.`,
-  sepaDebitTerms: _str =>
+  sepaDebitTerms: _ =>
```

Comment:
If the argument is not being used, please remove it altogether

Reply:
Updated